main.lua

```diff
@@ -86,6 +86,12 @@ end
 function setUpTitleScreen()
     menu:removeAllMenuItems()
     gfx.sprite.removeAll()
+    
+    if congratulationsTimer ~= nil then
+      removeGameData()
```

Comment:
so that if we setup title screen, but there was congratulation screen before, we clean saved state (e.g. button Continue will disappear since we finished the play)